proto/mobilecoind_mirror_api.proto

```diff
@@ -18,7 +19,9 @@ message QueryRequest {
     oneof request {
         GetProcessedBlockRequest get_processed_block = 1;
         mobilecoind_api.GetBlockRequest get_block = 2;
-        SignedRequest signed_request = 3;
+        mobilecoind_api.GetBlockInfoRequest get_block_info = 3;
+        google.protobuf.Empty get_ledger_info = 4;
+        SignedRequest signed_request = 5;
```

Comment:
If we are to make this breaking change, the better place for this would be the first tag (`= 1`) since we can't keep moving it every time we add new requests.

Reply:
Fixed in #7